src/pywatemsedem/geo/factory.py

```diff
@@ -280,7 +278,7 @@ def vector_factory(self, vector_input, geometry_type, allow_empty=False):
         elif isinstance(vector_input, gpd.GeoDataFrame):
             vector = VectorMemory(vector_input, geometry_type, allow_empty=allow_empty)
         else:
-            msg = "Input vector should be a geopandas GeoDataFrame or vector file."
+            msg = f"Input vector '{vector_input}'should be a geopandas GeoDataFrame or vector file."
```

Comment:
this may look weird if people send weird stuff, such as a normal pandas dataframe to this function. But anyway, I think it is helpful even in that case.